### PR TITLE
[sass/en] Add Sass tutorial

### PR DESCRIPTION
**Pull request by LauraNK**
Added Sass tutorial (written in SCSS syntax). 

Sass request in issue #1112

### CODE REVIEW

**Comment by vendethiel** [comment]
please add the markdown extension!

File: learnsass.scss [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+```	
+
+
+
+## SASS or Sass?
+Have you ever wondered whether Sass is an acronym or not? You probably haven't, but I'll tell you anyway. The name of the language is a word, "Sass", and not an acronym. 
+Because people were constantly writing it as "SASS", the creator of the language jokingly called it "Syntactically Awesome StyleSheets". 
+
+
+##Practice Sass
```
Comment (vendethiel) [inline learnsass.scss]: please space this comment

File: learnsass.scss [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+ul {
+  list-style-type: none;
+  margin-top: 2em;
+}
+
+ul li {
+  background-color: red;
+}
+
+ul li:hover {
```
Comment (vendethiel) [inline learnsass.scss]: maybe add a note here?

**Comment by LauraNK** [comment]
I wasn't sure what you meant by the last note. I assumed you meant adding a note to point out the ability to nest pseudo classes? 
I made the three changes, let me know if there's anything else :)

**Comment by vendethiel** [comment]
Fine by me.

**Comment by vendethiel** [comment]
:+1:

**Pull request merged**